kuma/static/styles/components/wiki/quick-links.scss

```diff
@@ -63,10 +63,10 @@
                 top: 1px;
             }
         }
+    }
 
-        a {
-            margin-bottom: 0;
-        }
+    summary a {
+        display: inline;
```

Comment:
This fixes the issue where the `<a>` text is slightly higher than text outside it.